docs/config.md

```diff
@@ -0,0 +1,163 @@
+# Configuration instruction
+
+You can see the default configuration [here](../devtools/config/docker_compose_config.toml).
+
+## Basic configuration
+
+### `center_id`
+
+The center id of the server. This is used to generate the distributed unique ID through snowflake algorithm.
+
+type: `u16`
+
+### `machine_id`
+
+The machine id of the server. This is used to generate the distributed unique ID through snowflake algorithm.
+
+type: `u16`
+
+### `indexer_mode`
+
+If mercury is in the indexer mode, mercury will pull blocks from the CKB node and append the block data to the database. Otherwise, mercury will only handle RPC requests and do not append the block data.
+
+type: `bool`
+
+### `need_sync`
+
+If this is true, mercury do will synchronize blocks parallelly when start. Otherwise, mercury will judge the necessity of synchronization parallel.
+
+type: `bool`
+
+### `rpc_thread_num`
+
+The thread number allocated for RPC server.
+
+type: `usize`
+
+### `flush_tx_pool_cache_interval`
+
+The millisecond interval for refreshing the transaction pool cache with the connected CKB node.
+
+type: `u64`
+
+### `cellbase_maturity`
+
+The epoch number of the cellbase maturity. This is same as the config of CKB. **DO NOT CHANGE THIS UNLESS TESTING**
+
+type: `u64`
+
+### `cheque_since`
+
+The epoch number of cheque cell can be withdraw. This is same as the data hard-coded in cheque lock script. **DO NOT CHANGE THIS UNLESS TESTING**
+
+type: `u64`
+
+## DB configuration
+
+### `max_connections`
+
+The maximum number of connections to database pool.
+
+type: `u32`
+
+### `db_type`
+
+The database type of Mercury. Such as `postgres`, `mysql`, `sqlite`.
```

Comment:
We only support postgres yet, you should mention this.

Reply:
Sure.